components/collector/ci/quality.sh

```diff
@@ -15,10 +15,9 @@ run mypy src
 run pylint --rcfile=../../.pylintrc src tests
 run python -m flake8 --select=DUO src  # Dlint
 unset PYTHONDEVMODE  # Suppress ResourceWarnings given by pip-audit in dev mode
-run pip-audit --strict --progress-spinner=off -r requirements/requirements-base.txt -r requirements/requirements.txt -r requirements/requirements-dev.txt
+run pip-audit --strict --progress-spinner=off -r requirements/requirements.txt -r requirements/requirements-dev.txt
```

Comment:
It looks like pip-audit also supports toml files, in fact the README.md says it *only* supports `pyproject.toml`

Reply:
It seems one can either tell pip-audit to audit the project (`pip-audit .`) where it reads the pyproject.toml file, but does not audit the optional dependencies or one can tell it to audit requirements files (`pip-audit -r requirements.txt`). As the requirements files also contain the indirect dependencies that seems to be the more complete option.